src/engine/Sound.ts

```diff
@@ -227,6 +263,11 @@ module ex.Internal {
       public onerror: (e: any) => void = () => { return; };
 
       public load() {
+         // Exit early if we already have data
+         if (!!this._data) {
```

Comment:
I considered it--sound would never evaluate to false here due to binary/uint data but it's still probably good.